cmd/starsd/cmd/init.go

```diff
@@ -100,19 +100,13 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 				}
 			}
 
-			if chainID == "big-bang-1" {
-				seeds = []string{
-					"69666f77b6a2355fcfc64c9879520a352b62917e@45.55.57.176:36656",
-				}
-			}
-
 			// Override default settings in config.toml
 			config.P2P.Seeds = strings.Join(seeds, ",")
 			config.P2P.MaxNumInboundPeers = 120
 			config.P2P.MaxNumOutboundPeers = 60
 			config.Mempool.Size = 10000
 			config.StateSync.TrustPeriod = 112 * time.Hour
-			config.FastSync.Version = "v0"
+			// config.FastSync.Version = "v0" // COMEBACK HERE
```

Comment:
I don't quite understand different versions since there is v1 as well

Reply:
I had commented it out as that property was not available under `config` anymore and didnt want to miss any configurations we have. 

Looks like its been renamed to BlockSync and only v0 is supported. v1 & v2 deprecated. https://github.com/tendermint/tendermint/pull/9259